README.md

```diff
@@ -131,6 +131,27 @@ kube-capacity --namespace-labels team=api
 kube-capacity --node-labels kubernetes.io/role=node
 ```
 
+### Filtering By Taints
```

Comment:
Thanks for adding docs to the readme, this is really great! One tiny suggestion would be to use "Node Taints" instead of just "Taints"
```suggestion
### Filtering By Node Taints
```